src/trading/getQuote.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  AccountAddress,
+  QuoteResults,
+  QuoterParameters,
+  SwapAdvancedSettings,
+  SwapParameters,
+  TradeParameters,
+} from './types'
+import { DEFAULT_QUOTE_VALIDITY, log } from './consts'
+
+import {
+  getQuoteAmountsAndCosts,
+  OrderBookApi,
+  OrderQuoteRequest,
+  OrderQuoteSideKindBuy,
+  OrderQuoteSideKindSell,
+  PriceQuality,
+  SigningScheme,
+} from '../order-book'
+import { buildAppData } from './appDataUtils'
+import { getOrderToSign } from './getOrderToSign'
+import { getIsEthFlowOrder, getSigner, swapParamsToLimitOrderParams } from './utils'
+import { Signer } from 'ethers'
+import { WRAPPED_NATIVE_CURRENCIES } from '../common'
+import { getOrderTypedData } from './getOrderTypedData'
+
+export type QuoteResultsWithSigner = { result: QuoteResults & { signer: Signer }; orderBookApi: OrderBookApi }
+
+export async function getQuote(
+  _tradeParameters: TradeParameters,
+  trader: QuoterParameters,
+  advancedSettings?: SwapAdvancedSettings,
+  _orderBookApi?: OrderBookApi
+): Promise<{ result: QuoteResults; orderBookApi: OrderBookApi }> {
+  const { appCode, chainId, account: from } = trader
+
+  const tradeParameters = getIsEthFlowOrder(_tradeParameters)
+    ? {
+        ..._tradeParameters,
+        sellToken: WRAPPED_NATIVE_CURRENCIES[chainId],
+      }
+    : _tradeParameters
```

Comment:
Just remembered we need more parameters when quoting an EthFlow order: https://github.com/cowprotocol/cowswap/blob/develop/apps/cowswap-frontend/src/api/cowProtocol/api.ts#L91-L99

Reply:
Great catch! Thanks!
https://github.com/cowprotocol/cow-sdk/pull/213/commits/61aad12d3453b585ca7c2be3e3b8d5b6000a77f9